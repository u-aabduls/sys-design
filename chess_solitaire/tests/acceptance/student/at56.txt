-- ACCEPTANCE TEST #56: VIOLATION CASE: KNIGHT BLOCKED - #1

-- Setup some chess pieces for the game
setup_chess (N,1,1)
setup_chess (B,2,1)
setup_chess (P,3,2)

start_game

move_and_capture(1,1,3,2)