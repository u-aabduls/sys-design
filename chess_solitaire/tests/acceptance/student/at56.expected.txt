  # of chess pieces on board: 0
  Game being Setup...
  ....
  ....
  ....
  ....
->setup_chess(N,1,1)
  # of chess pieces on board: 1
  Game being Setup...
  N...
  ....
  ....
  ....
->setup_chess(B,2,1)
  # of chess pieces on board: 2
  Game being Setup...
  N...
  B...
  ....
  ....
->setup_chess(P,3,2)
  # of chess pieces on board: 3
  Game being Setup...
  N...
  B...
  .P..
  ....
->start_game
  # of chess pieces on board: 3
  Game In Progress...
  N...
  B...
  .P..
  ....
->move_and_capture(1,1,3,2)
  # of chess pieces on board: 3
  Error: Block exists between (1, 1) and (3, 2)
  N...
  B...
  .P..
  ....
